pages/learn/develop/integrations.md

```diff
@@ -5,11 +5,11 @@ excerpt: Guides to integrate {{ $names.company.lower }} with a variety of IoT pl
 
 # Integrate with 3rd parties
```

Comment:
@kb2ma Why do we even have this page, and then the `cloud-iot-provisioning.md` file in a directory. I think this document doesn't provide no value? Rather this intro could be added to the other file instead

Reply:
Thanks for the fresh eyes on this. I agree there isn't sufficient purpose for a separate 3rd party integrations page. Created a "Provision with cloud IoT" navigation entry, which links directly to `cloud-iot-provisioning.md`. Also removed the `integrations` directory. See fd5f464.